core/block.ex

```diff
@@ -2,16 +2,15 @@ defmodule UltraDark.Blockchain.Block do
   alias UltraDark.Blockchain.Block
   alias UltraDark.Utilities
   alias UltraDark.Transaction
-  defstruct [
-    index: nil,
-    hash: nil,
-    previous_hash: nil,
-    difficulty: nil,
-    nonce: 0,
-    timestamp: nil,
-    merkle_root: nil,
-    transactions: []
-  ]
+
+  defstruct index: nil,
+            hash: nil,
+            previous_hash: nil,
+            difficulty: nil,
+            nonce: 0,
+            timestamp: nil,
+            merkle_root: nil,
+            transactions: []
```

Comment:
if we're gonna rewrite it like this, lets align all the values with the one on the top